packages/ember-runtime/lib/inject.js

```diff
@@ -10,7 +10,10 @@ import { assert } from 'ember-debug';
   @public
 */
 export default function inject() {
-  assert(`Injected properties must be created through helpers, see '${Object.keys(inject).join('"', '"')}'`);
+  const helpers = Object.keys(inject)
```

Comment:
I think this should inlined inside `assert` else it won't be stripped

Reply:
I wondered about that, and can push an update to that effect tomorrow.

Reply:
That would solve the linting problem as well. Thank you!

Reply:
Updated!